src/core/server/ui_settings/utils.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { UiSettingScope } from './types';
+
+export const CURRENT_USER_PLACEHOLDER = '<current_user>';
+
+export const generateDocId = (id: string, scope?: UiSettingScope) => {
```

Comment:
This function name is not inline with what it is actually doing.

Reply:
updated to `buildDocIdWithScope`